docs/articles/vault.md

```diff
@@ -1,43 +1,194 @@
 # Using Vault
 
-Testkube has not been verified to work with the various ways Vault can be
-integrated into a Kubernetes cluster, but we are ready to support enterprise
-customers with the specifics of their environment.
+Testkube should be able to integrate with the various ways Vault integrates into
+a Kuberentes cluster. This guide specifically covers the use of the [sidecar
+injector](https://developer.hashicorp.com/vault/docs/platform/k8s/injector), but
+we are ready to support enterprise customers utilizing the secrets operator or
+the CSI provider.
 
-For integrations utilizing the [sidecar
-injector](https://developer.hashicorp.com/vault/docs/platform/k8s/injector) you
-can try to set the appropriate annotations by adapting the example
-configurations below for your needs. If you encounter issues please reach out to
-our enterprise support.
+:::warning
 
-## Configurations for sidecar injector
+Prebuilt and container executors are deprecated so we do not officially support
+their usage with Vault, please migrate to workflows.
```

Comment:
Should there be a link here to the docs to migrate to workflows?

Reply:
Could you point me to the article, I am not able to find it.

Reply:
https://docs-v2.testkube.io/articles/test-workflow-migration